lang/ruby/ruby.def

```diff
@@ -0,0 +1,5 @@
+Bootstrap: docker
+From: ruby:2.6-rc-stretch
+
+%runscript
+"$@"
```

Comment:
Why did you do it this way? Instead of having something like:

```
%runscript
    exec ruby "$@"
```

Reply:
So you can run a Ruby script by: `./ruby.sif ruby hello-world.rb` or: `./ruby.sif ./hello-world.rb`.